### PR TITLE
Load log files using new method and not file annotation.

### PR DESCRIPTION
**Pull request by jburel**
This method will only be used if the log file ID was not correctly returned by the import event.

### CODE REVIEW

File: components/insight/SRC/org/openmicroscopy/shoola/env/data/OmeroMetadataService.java [hunk 1]
```diff
@@ -734,4 +735,18 @@ public RequestCallback downloadMetadataFile(SecurityContext ctx, File file,
...
+	/**
+	 * Loads the log files linked to the specified objects.
+	 *
+	 * @param ctx The security context.
+	 * @param rootType The type of object to handle.
+	 * @param rootIDs The collection of object's identifiers.
+	 * @return See above.
+	 * @throws DSOutOfServiceException If the connection is broken, or logged in
+     * @throws DSAccessException If an error occurred while trying to 
+     * retrieve data from OMERO service.
```
Comment (mtbc) [inline components/insight/SRC/org/openmicroscopy/shoola/env/data/OmeroMetadataService.java]: whitespace strange

File: components/insight/SRC/org/openmicroscopy/shoola/env/data/views/calls/FileUploader.java [hunk 2]
```diff
@@ -30,6 +30,10 @@
 import java.util.List;
 import java.util.Map;
 
+import omero.model.IObject;
+import omero.model.OriginalFile;
```
Comment (mtbc) [inline components/insight/SRC/org/openmicroscopy/shoola/env/data/views/calls/FileUploader.java:33]: application-internal

File: components/insight/SRC/org/openmicroscopy/shoola/env/data/views/calls/FileUploader.java [hunk 3]
```diff
@@ -30,6 +30,10 @@
...
+
+import org.apache.commons.collections.CollectionUtils;
```
Comment (mtbc) [inline components/insight/SRC/org/openmicroscopy/shoola/env/data/views/calls/FileUploader.java]: third-party

File: components/insight/SRC/org/openmicroscopy/shoola/env/data/OMEROGateway.java [hunk 4]
```diff
@@ -8337,4 +8337,30 @@ Roles getSystemRoles(SecurityContext ctx)
         }
         return null;
      }
+
```
Comment (mtbc) [inline components/insight/SRC/org/openmicroscopy/shoola/env/data/OMEROGateway.java:8340]: copyright to 2014

**Comment by jburel** [comment]
@mtbc: done

**Pull request merged**